yt_dlp/extractor/tenplay.py

```diff
@@ -66,42 +63,21 @@ class TenPlayIE(InfoExtractor):
         'X': 18,
     }
 
-    def _get_bearer_token(self, video_id):
```

Comment:
I assume you removed this due to login being no longer required. Is this the case for all content?

Reply:
Yes.